Sources/NIOHTTPClient/HTTPHandler.swift

```diff
@@ -207,7 +207,7 @@ internal extension URL {
 
 public extension HTTPClient {
     final class Task<Response> {
-        let future: EventLoopFuture<Response>
+        public let future: EventLoopFuture<Response>
```

Comment:
This issue will be addressed here https://github.com/swift-server/swift-nio-http-client/pull/37/files#diff-4345dbdc9050b28bde080d64fa672bebR232